api/SVGStopElement.json

```diff
@@ -0,0 +1,52 @@
+{
+  "api": {
+    "SVGStopElement": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/SVGStopElement",
+        "support": {
+          "webview_android": {
+            "version_added": false
+          },
+          "chrome": {
+            "version_added": null
+          },
+          "chrome_android": {
+            "version_added": null
```

Comment:
All 3 flavors of Chrome should be `true`.

https://cs.chromium.org/chromium/src/third_party/blink/renderer/core/svg/svg_stop_element.idl